obt verify currently broken

A fresh install of all the latest dependencies produces this…  Started noticing this on Friday…
```
obt verify
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: Invalid non-string/buffer chunk
    at chunkInvalid (/Users/mandrews/sandboxes/next-ads-component/node_modules/origami-build-tools/node_modules/stream-combiner2/node_modules/readable-stream/lib/_stream_readable.js:421:10)
    at readableAddChunk (/Users/mandrews/sandboxes/next-ads-component/node_modules/origami-build-tools/node_modules/stream-combiner2/node_modules/readable-stream/lib/_stream_readable.js:176:12)
    at DuplexWrapper.Readable.push (/Users/mandrews/sandboxes/next-ads-component/node_modules/origami-build-tools/node_modules/stream-combiner2/node_modules/readable-stream/lib/_stream_readable.js:162:10)
    at Transform.<anonymous> (/Users/mandrews/sandboxes/next-ads-component/node_modules/origami-build-tools/node_modules/stream-combiner2/node_modules/duplexer2/index.js:50:15)
    at emitOne (events.js:77:13)
    at Transform.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Transform.Readable.push (_stream_readable.js:110:10)
    at Transform.push (_stream_transform.js:128:32)
    at afterTransform (_stream_transform.js:76:12)
make: *** [test] Error 1
```

--- a/lib/tasks/verify.js
+++ b/lib/tasks/verify.js
@@ -158,7 +158,7 @@ module.exports.esLint = function(gulp, config) {
 	// Exclude files specified in config.excludeFiles
 	excludeFiles = excludeFiles.concat(config.excludeFiles || []);
 
-	const combinedStream = combine(
+	const combinedStream = combine.obj(
 		gulp.src(['**/*.js'].concat(excludeFiles)),
 		eslint({
 			configFile: configPath,

--- a/package.json
+++ b/package.json
@@ -48,7 +48,7 @@
     "npmconf": "^2.1.2",
     "portfinder": "^0.4.0",
     "semver": "^4.3.6",
-    "stream-combiner2": "1.0.2",
+    "stream-combiner2": "^1.0.2",
     "through2": "^2.0.0",
     "vinyl-buffer": "^1.0.0",
     "vinyl-source-stream": "^1.1.0",

